### ISSUE TITLE
M1.5: Add warning to the test "Incorrect Query with configuration using only specifickey with wrong value type"

### ISSUE THREAD

**Issue by CamiloBorba09** [montevideo-tech/cmcd-validator#113]
We need to add a warning in this test case. 

```
{

    description:
      "Test: Incorrect Query with configuration using only specifickey with wrong value type",
    query:
      "https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps_3840x2160_12000k/bbb_30fps_3840x2160_12000k_0.m4v?CMCD=qualabs-br%3Dfalse%2Ccid%3D%2221cf726cfe3d937b5f974f72bb5bd06a%22%2Cot%3Di%2Csf%3Dd%2Csid%3D%22b248658d-1d1a-4039-91d0-8c08ba597da5%22%2Cst%3Dv%2Csu%3D23",
    config: {
      specificKey: ["su", "sid"],
    },
    output: {
      valid: false,
      errors: [
        {
          description: "Value type is incorrect",
          key: "su",
          type: "wrong-type-value",
          value: "23",
        },
      ],
      warnings: [],
      parsedData: undefined,
      rawData:
        "https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps_3840x2160_12000k/bbb_30fps_3840x2160_12000k_0.m4v?CMCD=qualabs-br%3Dfalse%2Ccid%3D%2221cf726cfe3d937b5f974f72bb5bd06a%22%2Cot%3Di%2Csf%3Dd%2Csid%3D%22b248658d-1d1a-4039-91d0-8c08ba597da5%22%2Cst%3Dv%2Csu%3D23",
    },
}

```
Although in this case the library must report errors of the **su** and **sid** keys, it should also report a warning since the **qualabs-br** key does not belong to the standard.

**Comment by CamiloBorba09** [comment]
This ticket is unblocked since #126 is done

**Comment by adriansqualabs** [comment]
Maybe we need to think again over this issue, when we solve #126 ticket, the warning with the customKey declared is now working.

**Comment by CamiloBorba09** [comment]
We discuss with the team and we decided no to do this because we think that the library only need to report errors and warnings of the specific key.
